map.js

```diff
@@ -0,0 +1,63 @@
+import dinosaurs from "./dinosaurs.json" assert { type: "json" };
+console.log(dinosaurs)
+
+const dinoContainer = document.getElementById('flip-card');
```

Comment:
This puts all of the dino cards in a single "flip-card" div.  See index.html line 123.  That is why all of the cards flip on .flip-card:hover (styles.css line 284).

Go back to making a container div for this const dinoContainer.  Then each dino card needs its own "flip-card" class.  Then each individual card will flip on hover.